src/main/java/redis/clients/jedis/params/GeoRadiusParam.java

```diff
@@ -58,6 +60,17 @@ public GeoRadiusParam count(int count) {
     return this;
   }
 
+  public GeoRadiusParam count(int count, boolean any) {
```

Comment:
This is a new feature. Add proper unit tests for it.

Reply:
Added [here](https://github.com/AvitalFineRedis/jedis/pull/1)

Reply:
Added [here](https://github.com/AvitalFineRedis/jedis/pull/1)